app/models/payment/go_cardless/subscription.rb

```diff
@@ -2,16 +2,16 @@ class Payment::GoCardless::Subscription < ActiveRecord::Base
   class Charge
     attr_reader :subscription, :event
 
-    delegate :page, :amount, :currency, to: :subscription
+    delegate :page_id, :amount, :currency, :payment_method_id, :customer_id, to: :subscription
 
     def initialize(subscription, event = {})
       @event = event
       @subscription = subscription
     end
 
     def call
-      Payment::GoCardless.write_transaction(event['links']['payment'], amount, currency, page.id, subscription)
-
+      Payment::GoCardless.write_transaction(event['links']['payment'], amount, currency,
+        Date.today, page_id, customer_id, payment_method_id, subscription)
```

Comment:
Why is this using `Date.today`? GBP charges should happen 20th of the next available month.